torch/distributed/rendezvous.py

```diff
@@ -101,18 +101,18 @@ def _error(msg):
 
     if url != "env://":
         raise _error("Only `env://` is expected for the env init method")
-    world_size = os.environ["WORLD_SIZE"]
+    world_size = os.environ.get("WORLD_SIZE", None)
     if world_size is None:
-        raise _error("world size is missing")
-    rank = os.environ["RANK"]
+        raise _error("WORLD_SIZE not set")
```

Comment:
nit: once and for all, improving this error message would be nice.
For example, getting this error is much better than before

```
RuntimeError: env:// rendezvous: WORLD_SIZE not set
```

But can be greatly improved:

```
RuntimeError: While initializing torch.distributed using the env:// rendezvous mechanism: environment variable WORLD_SIZE expected, but it is not set.
```

Reply:
Good point @soumith, updating.